### ISSUE TITLE
Some parts of the documentation are outdated

### ISSUE THREAD

**Issue by pedroota** [prevwong/craft.js#533]
I recently noticed that some parts of docs are outdated and some missing pieces such as examples that the docs puts. Someone having this problem?

**Comment by akucinskii** [comment]
Yup, some features are missing and some mentioned in the documentation are no longer available.
